backend/src/models/question.js

```diff
@@ -18,7 +19,10 @@ class Question {
                 {
                     [QUESTIONS]: { id }
                 })
-            .then(([row]) => row || {})
+            .then(([row]) => {
+                if (row) return row
+                throw new APIError({ status: 404, code: 4001, message: 'The question does not exist' })
```

Comment:
Be good to have less magic numbers. eg what is 4001?

Reply:
Well it's the error code.
I'm not sure the best way to do this tbh - maybe we should have a constant for each error code? In this case it would be something like `ERR_QUESTION_NOTEXIST` ?